sources/web/datalab/polymer/modules/GapiManager.ts

```diff
@@ -34,7 +40,8 @@ class GapiManager {
     // Loads the gapi client library and the auth2 library together for efficiency.
     // Loading the auth2 library is optional here since `gapi.client.init` function will load
     // it if not already loaded. Loading it upfront can save one network request.
-    GapiManager._loadClientId()
+    console.log('== loading gapi');
+    return GapiManager._loadClientId()
```

Comment:
Yes this was my proposition, I'm also not a huge fan of tying the type system to the naming, but I think it's useful to quickly know the logic flow as I'm calling the function.
How about we stick to an explicit function return type then?